.travis.yml

```diff
@@ -34,8 +34,8 @@ before_install:
   hash -r &&
   conda config --set always_yes yes --set changeps1 no &&
   conda update -q conda &&
-  conda install python=${TRAVIS_PYTHON_VERSION} &&
   conda config --add channels conda-forge &&
+  conda install python=${TRAVIS_PYTHON_VERSION} &&
```

Comment:
now that we can install almost any python dependencies with pip, would it make sense to simplify this for the time being to just installing with `pip -y -r requirements.txt` instead of using conda install in the next step.